### DISCUSSION TITLE
Release v1.0.1

### DISCUSSION THREAD

**Discussion by skyleaworlder** [TJ-CSCCG/TongjiThesis#18] [Announcements]
* 适配页眉页脚横线粗细、字体、行距、矢量 Logo、致谢标题、摘要间距；
* 引入代码块盒子、交叉引用、长表格等功能性宏包；
* 增加毕业设计论文（理工类）模板封面；
* 改变项目结构；
* 增加 Windows 下编译脚本、VSCode LaTeX-Workshop 的配置文件；
* 增加 MacOS / Linux / Windows 三平台的 workflow。

<hr /><em>This discussion was created from the release <a href='https://github.com/TJ-CSCCG/tongji-undergrad-thesis/releases/tag/v1.0.1'>Release v1.0.1</a>.</em>

**Comment by RizhongLin** [comment]
Great things come from hard work and perseverance. ❤️👍

**Reply by skyleaworlder** [reply]
Thanks for your appreciation! (But I fail to realize that you like to quote famous sayings in English, not only in Chinese. 🤣
